app/models/action.rb

```diff
@@ -3,5 +3,13 @@ class Action < ActiveRecord::Base
   belongs_to :member
 
   has_paper_trail on: [:update, :destroy]
+
+  after_create :update_member_donor_status
```

Comment:
Classic rails an anti-pattern, modifying another model from an AR callback. This doesn't have to happen every time an action is created. What about placing this logic in its right context? One place that could work: https://github.com/SumOfUs/Champaign/blob/development/app/controllers/payment_controller.rb#L14